index.html

```diff
@@ -35,6 +35,35 @@
   </nav>
 </header>
 
+
+  <div class="mobile-nav d-flex" id="mobile-nav">    
```

Comment:
- [ ] Your mobile menu does not still fit the screen as can be seen in the provided Figma template. Kindly ensure to improve upon this aspect as also requested by the previous code reviewer.

**Your mobile design:**
![image](https://user-images.githubusercontent.com/12745474/131401410-cce90b67-a185-4e84-9d66-81d343280e33.png)


The original mobile menu design:
![image](https://user-images.githubusercontent.com/52497246/131392836-5be4a251-5723-471c-b8f6-5956e8178bf0.png)